virtusize-core/src/main/java/com/virtusize/android/network/VirtusizeApiTask.kt

```diff
@@ -38,8 +38,9 @@ class VirtusizeApiTask(
         private val CONNECT_TIMEOUT = TimeUnit.SECONDS.toMillis(60).toInt()
 
         // The request header keys
-        private const val HEADER_BROWSER_ID = "x-vs-bid"
-        private const val HEADER_AUTH = "x-vs-auth"
+        private const val HEADER_VS_BROWSER_ID = "x-vs-bid"
+        private const val HEADER_VS_AUTH = "x-vs-auth"
+        private const val HEADER_VS_STORE_ID = "x-vs-store-id"
```

Comment:
Without setting this request header `x-vs-store-id`, sometimes the API endpoint /user-body-measurements returns a different user body than the one from Aoyama.
This fixes 
https://app.clickup.com/t/3702259/NSDK-241

Reply:
This is because `"x-vs-store-id` is used with `x-vs-external-user-id` to create a uniq key in the backend..